app/controllers/bulk_discounts_controller.rb

```diff
@@ -25,10 +25,12 @@ def create
   end
 
   def destroy
+    # require 'pry';binding.pry
+    merchant = Merchant.find(params[:merchant_id])
```

Comment:
This needs to be [:merchant_id] otherwise the id is not found in the bulk_discounts_controller.